bin/autoless

```diff
@@ -16,6 +16,7 @@ program
   .option('--no-notify', "Do not send any notifications")
   .option('--no-sound', "Do not play a sound with error notifications")
   .option('--source-map', "Generate source map files next to css files")
+  .option('--compress', "Compress output by removing some whitespaces")
```

Comment:
Can we change this to "Compress output by removing whitespace"?

http://english.stackexchange.com/a/25371